resources/js/components/ui/Table/TQueryTable.tsx

```diff
@@ -145,13 +158,15 @@ export const TQueryTable: VoidComponent<TQueryTableProps> = (props) => {
     ["uuid", {cell: tableCells.uuid, enableSorting: false, size: 80}],
   ]);
 
+  const [allInited, setAllInited] = createSignal(false);
```

Comment:
ok, sprawdziłem, że to poprawne, ale wciąż initialized to powszechniejsza forma i myślę ze lepsza dla czytelności.

Reply:
No dooobra.